### PR TITLE
testing of services, ORM schema fixing

### PR DESCRIPTION
**Pull request by kukuruzavictor**

### CODE REVIEW

File: src/test/java/com/feedback/service/FeedbackRequestServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.openMocks;
+
+class FeedbackRequestServiceTest {
+
+    @Mock
+    FeedbackRequestRepo feedbackRequestRepo;
```
Comment (Forfend) [inline src/test/java/com/feedback/service/FeedbackRequestServiceTest.java:38]: Please, add access modifier

File: src/main/java/com/feedback/controller/CourseController.java [hunk 2]
```diff
@@ -14,6 +15,7 @@
 
 @RestController
 @RequestMapping("/courses")
+@CrossOrigin(origins = "http://localhost:4200")
```
Comment (Forfend) [inline src/main/java/com/feedback/controller/CourseController.java:18]: It should be in configs

**Review by Forfend — CHANGES_REQUESTED** [review]

File: src/test/java/com/feedback/service/FeedbackRequestServiceTest.java (hunk 1 above)
Comment (kukuruzavictor) [reply src/test/java/com/feedback/service/FeedbackRequestServiceTest.java]: Fixed

File: src/main/java/com/feedback/controller/CourseController.java (hunk 2 above)
Comment (kukuruzavictor) [reply src/main/java/com/feedback/controller/CourseController.java]: Didn’t found the way to put the link to app.properties. Just removed to open gate for all ports. I believe it’s not a big deal on our test project.
PS. “endpoints.cors.allowed-origins” not works

File: src/main/java/com/feedback/controller/FeedbackRequestController.java [hunk 3]
```diff
@@ -2,13 +2,15 @@
...
 import com.feedback.service.FeedbackRequestService;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequestMapping("/")
+@CrossOrigin
```
Comment (Forfend) [inline src/main/java/com/feedback/controller/FeedbackRequestController.java:13]: I suggest adding the Global CORS configuration. 
Reference: https://spring.io/blog/2015/06/08/cors-support-in-spring-framework

**Review by Forfend — CHANGES_REQUESTED** [review]

File: src/main/java/com/feedback/controller/FeedbackRequestController.java (hunk 3 above)
Comment (kukuruzavictor) [reply src/main/java/com/feedback/controller/FeedbackRequestController.java:13]: Resolved as suggested.

**Review by Forfend — APPROVED** [review]

**Pull request merged**